host_applications/linux/apps/gencmd/gencmd.c

```diff
@@ -71,6 +86,14 @@ int main( int argc, char **argv )
 
     if (argc > 1)
     {
+       // first check if we were invoked with either -? or --help
+       // in which case show basic usage and exit
+      if( ( strcmp( argv[1], "-?" ) == 0) || strcmp( argv[1], "--help" ) == 0 )
```

Comment:
I've definitely seen `--help` used before, but is `-?` a common flag for Linux CLI programs?

Reply:
Well I've seen it used and it has 2 LGTMs from reviewers so I'm happy to leave it in unless @popcornmix objects

Reply:
-h is more usual.

Reply:
I will change it to -h in a future PR, since I'm going to be creating one to have it output errors to STDERR anyway and I've already squashed my commits down. Unless anyone has any objections?

Reply:
Changed -? to -h